src/rembhandler.cpp

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2024 Vladimir Voronin
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+#include "rembhandler.hpp"
+#include "rtp.hpp"
+
+#ifdef _WIN32
+#include <winsock2.h>
+#else
+#include <arpa/inet.h>
+#endif
+
+#if RTC_ENABLE_MEDIA
+
+namespace rtc {
+
+RembHandler::RembHandler(std::function<void(unsigned int, unsigned int)> onRemb) : mOnRemb(onRemb) {}
+
+void RembHandler::incoming(message_vector &messages, [[maybe_unused]] const message_callback &send) {
+	for (const auto &message : messages) {
+		size_t offset = 0;
+		while ((sizeof(RtcpHeader) + offset) <= message->size()) {
+			auto header = reinterpret_cast<RtcpHeader *>(message->data() + offset);
+			uint8_t payload_type = header->payloadType();
+
+			if (payload_type == 206 && header->reportCount() == 15) {
+                auto remb = reinterpret_cast<RtcpRemb *>(message->data() + offset);
+                mOnRemb(remb->getNumSSRC(), remb->getBitrate());
```

Comment:
Why do you pass the number of SSRCs (instead of the SSRCs themselves) in the REMB feedback message? It seems useless for the user. I think you should remove it from the callback signature. If I understand correctly, it is not necessary to pass SSRCs as the bitrate is the total bitrate for the RTP session anyway. The SSRCs should be used for dispatching across tracks in `PeerConnection::dispatchMedia()`, if necessary.

Reply:
Maybe. i don't understand well goog-remb, i think maybe its need for anything, i will change.